services/reconciler/route53-client.server.ts

```diff
@@ -126,7 +127,7 @@ export const getDnsRecordSetPage = async (
     const command = new ListResourceRecordSetsCommand({
       HostedZoneId: await getHostedZoneId(),
       StartRecordName: fqdn,
-      StartRecordType: type,
+      StartRecordType: type ? toRoute53RRType(type) : undefined,
```

Comment:
Can we change this function so that the `type` parameter is of type `RRType`? Seems like this comment doesn't apply anymore.

```js
export const getDnsRecordSetPage = async (
  fqdn?: string,
  // Using string as it cam be any record type not just the ones we use, also AWS sdk refers to it as string
  type?: string
)```